### PR TITLE
🚢 Refactor run_python to make it more testable

### PR DESCRIPTION
**Pull request by davidkircos**
Moves processing output into it's own function to make it possible to test

- [x] move logic to a function
- [x] write tests

### CODE REVIEW

File: quadratic-kernels/python-wasm/quadratic_py/process_output.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    output_type = type(output_value).__name__
+    output_size = None
+
+    # TODO(ddimaria): figure out if we need to covert back to a list for array_output
+    # We should have a single output
+    if isinstance(output_value, list):
+        if len(output_value) > 0:
+            array_output = output_value
+            if (isinstance(output_value[0], list)):
+                output_size = (len(output_value[0]), len(output_value))
```
Comment (davidkircos) [inline quadratic-kernels/python-wasm/quadratic_py/process_output.py:18]: Fixed a bug here with 2d array sizing

Comment (ddimaria) [reply quadratic-kernels/python-wasm/quadratic_py/process_output.py:18]: We lost the diff.  Is this the only change in this code?

Comment (davidkircos) [reply quadratic-kernels/python-wasm/quadratic_py/process_output.py:18]: yes

**Review by ddimaria — APPROVED** [review]
Great test additions!  Python testing is improving greatly!

**Pull request merged**